pkg/log/logcontext_test.go

```diff
@@ -0,0 +1,155 @@
+package log
+
+import (
+	"bytes"
+	"context"
+	"net/http"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/mocks"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/google/uuid"
+	"github.com/sirupsen/logrus"
+
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/go-openapi/runtime/security"
+
+	"github.com/openshift/assisted-service/client"
+	clientInstaller "github.com/openshift/assisted-service/client/installer"
+	"github.com/openshift/assisted-service/restapi"
+	"github.com/openshift/assisted-service/restapi/operations/installer"
+)
+
+//var _ restapi.InstallerAPI = FakeInventory{}
+
+func serv(server *http.Server) {
+	_ = server.ListenAndServe()
+}
+
+type AuthPayload struct {
+	Username     string `json:"username"`
+	FirstName    string `json:"first_name"`
+	LastName     string `json:"last_name"`
+	Organization string `json:"org_id"`
+	Email        string `json:"email"`
+	Issuer       string `json:"iss"`
+	ClientID     string `json:"clientId"`
+	IsAdmin      bool   `json:"is_admin"`
+	IsUser       bool   `json:"is_user"`
+}
+
+var testUser = AuthPayload{
+	IsUser:   true,
+	Username: "dummy",
+}
+
+var authenticator = func(name string, _ string, authenticate security.TokenAuthentication) runtime.Authenticator {
+	return security.HttpAuthenticator(func(r *http.Request) (bool, interface{}, error) {
+		return true, &testUser, nil
+	})
+}
+
```

Comment:
Why is this needed in the context of the log test?
We are not testing authentication

Reply:
Since the flow go trough the server I need to disable authentication. However, since the tests are in the log package - which is a low level package - I could not import the auth package and disable authentication in more reasonable way. The solution was these callbacks that do the job. Either way there would have been some lines in the tests that meant to disable auth